src/main/java/org/cactoos/scalar/Reduced.java

```diff
@@ -84,6 +84,21 @@
      */
     private final BiFunc<T, T, T> function;
 
+    /**
+     * Ctor.
+     * @param values The scalars
+     * @param reduce Reducing function
+     */
+    public Reduced(
```

Comment:
@andreoss I blieve this is out of scope of this PR now, no?

Reply:
Removed